src/Model/PixelEvent.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusFacebookPlugin\Model;
+
+use DateTimeInterface;
+use Setono\ClientId\ClientId;
+use Setono\SyliusFacebookPlugin\ServerSide\ServerSideEventInterface;
+use Symfony\Component\Uid\Uuid;
+
+class PixelEvent implements PixelEventInterface
+{
+    protected string $id;
+
+    protected ?PixelInterface $pixel = null;
+
+    protected ?ClientId $clientId = null;
+
+    protected bool $consentGranted = false;
+
+    protected string $eventName = ServerSideEventInterface::EVENT_VIEW_CONTENT;
```

Comment:
I don't think we should have a default value for this?

Reply:
It filled automatically by factory, so no problem with that.
Default value provided to make var non-nullable.

Reply:
Yeah, but it's still a wrong default value imo, so the best approach would be to make it nullable imo